features/admin.applications.v1/pages/applications.tsx

```diff
@@ -117,7 +117,8 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
 ): ReactElement => {
 
     const {
-        [ "data-testid" ]: testId
+        [ "data-testid" ]: testId,
```

Comment:
`data-testid` is not required

Reply:
`data-testid` was already there and used by Applications page.